### DISCUSSION TITLE
Add BelkovskiyS to the feed

### DISCUSSION THREAD

**Discussion by algot** [umputun/feed-master#118] [General]
Could you please add the new channel https://www.youtube.com/@BelkovskiyS to the feed?

**Comment by umputun** [comment]
I have moved to discussions because this is not a bug report. Pls note - this repository is not specific to any particular set of feeds but rather for a generic service allowing any feeds and sources.  Requests/ideas like this one are better to be sent to me directly via email or tg.

Anyhow, regarding your request - it is too early as the feed is brand new, and I don't know how often it will have new episodes, the length of the episode, and so on. Well, I even wonder if any other episodes will be published. All this info is needed to decide if it is worth the effort, disk space, traffic, how many episodes to keep, etc.
